rules/ast/ast.go

```diff
@@ -732,7 +743,10 @@ func evalVectorBinop(opType BinOpType,
 	case And:
 		return lhs, true
 	case Or:
-		return lhs, true // TODO: implement OR
+		if lhs == 0 {
```

Comment:
Your first comment was correct - thought your second one was referring to having `1 or 2` in the QL, which isn't the case.
Guess you were referring to the internals?